docs/section23/Rule23-5.md

```diff
@@ -0,0 +1,48 @@
+
+# Airside System - Rule 23-5  
+
+**Rule ID:** 23-5  
+**Rule Description:** For baseline systems 6 and 8, Fans in parallel VAV fan-powered boxes shall be sized for 50% of the peak design primary air (from the VAV air-handling unit) flow rate and shall be modeled with 0.35 W/cfm fan power.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**Appendix G Section Reference:** Section G3.1.3.14 Fan Power and Control (Systems 6 and 8)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-6 or 8  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. baseline_hvac_type()  
+
+## Rule Logic:  
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - For each terminal in zone: `for terminal_b in zone_b.terminals:`
+
+    - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+  
+      - Check if HVAC system is type 6 or 8: `if baseline_hvac_type(hvac_b) in ["SYS-6", "SYS-8"]:`
+
```

Comment:
based on MK's response to 23-1, my understanding is that line 23 is referencing a function to be written in the future, which will identify the baseline system type for each HVAC system in the B-RMR.